R/pkg/R/functions.R

```diff
@@ -86,6 +86,22 @@ NULL
 #' df <- createDataFrame(data.frame(time = as.POSIXct(dts), y = y))}
 NULL
 
+#' String functions for Column operations
+#'
+#' String functions defined for \code{Column}.
+#'
+#' @param x Column to compute on. In \code{instr}, it is the substring to check. In \code{format_number},
```

Comment:
In the string functions, `instr` and `format_number` are the only two methods that have the `(y, x)` signature. And yes, there was doc on the `y` parameter down below. Now I bring it up and document here.